sdks/java/extensions/sql/src/test/java/org/apache/beam/sdk/extensions/sql/meta/provider/mongodb/MongoDbReadWriteIT.java

```diff
@@ -87,24 +104,51 @@
   private static final String collection = "collection";
   private static MongoDBPipelineOptions options;
 
+  @ClassRule public static final TemporaryFolder MONGODB_LOCATION = new TemporaryFolder();
+
+  private static final MongodStarter mongodStarter = MongodStarter.getDefaultInstance();
+  private static MongodExecutable mongodExecutable;
+  private static MongodProcess mongodProcess;
+  private static MongoClient client;
+
   @Rule public final TestPipeline writePipeline = TestPipeline.create();
   @Rule public final TestPipeline readPipeline = TestPipeline.create();
 
   @BeforeClass
   public static void setUp() throws Exception {
+    int port = NetworkTestHelper.getAvailableLocalPort();
+    LOG.info("Starting MongoDB embedded instance on {}", port);
+    IMongodConfig mongodConfig =
+        new MongodConfigBuilder()
+            .version(Version.Main.PRODUCTION)
+            .configServer(false)
+            .replication(new Storage(MONGODB_LOCATION.getRoot().getPath(), null, 0))
+            .net(new Net("localhost", port, Network.localhostIsIPv6()))
+            .cmdOptions(
+                new MongoCmdOptionsBuilder()
+                    .syncDelay(10)
+                    .useNoPrealloc(true)
+                    .useSmallFiles(true)
+                    .useNoJournal(true)
+                    .verbose(false)
+                    .build())
+            .build();
+    mongodExecutable = mongodStarter.prepare(mongodConfig);
+    mongodProcess = mongodExecutable.start();
+    client = new MongoClient("localhost", port);
+
     PipelineOptionsFactory.register(MongoDBPipelineOptions.class);
     options = TestPipeline.testingPipelineOptions().as(MongoDBPipelineOptions.class);
+    options.setMongoDBHostName("localhost");
+    options.setMongoDBPort(port);
```

Comment:
Can you update the docstring as well? It still says you can point to your own instance, but it looks like thats not an option anymore.

Reply:
Yes, it's not an option anymore to pass pipeline options anymore. Updated the JavaDoc string and got rid of pipeline options.